src/transformers/tokenization_auto.py

```diff
@@ -217,6 +217,17 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
             return BertJapaneseTokenizer.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
 
         use_fast = kwargs.pop("use_fast", False)
+
+        if config.tokenizer_class is not None:
+            if use_fast and not config.tokenizer_class.endswith("Fast"):
+                tokenizer_class_candidate = f"{config.tokenizer_class}Fast"
+            else:
+                tokenizer_class_candidate = config.tokenizer_class
+            tokenizer_class = globals().get(tokenizer_class_candidate)
```

Comment:
Might be cleaner to use some of our internal list/dict containing all tokenizers, just in case there are weird things in the namespace of some users.

Reply:
Yes I was wondering about that. I was wondering if by using `globals()` someone could even use a tokenizer that's not in the library, but I don't think so, as `globals` is actually locals in this scope/file if I understand correctly.